src/components/flow-builder/edge-types/workflow-edge.tsx

```diff
@@ -48,16 +51,18 @@ export default function CustomEdge({
         d={edgePath}
         markerEnd={markerEnd}
       />
-      <EdgeLabelRenderer>
-        <div
-          className="nodrag nopan rounded-badge pointer-events-auto absolute flex h-10 w-10 items-center justify-center bg-base-content"
-          style={{
-            transform: `translate(-50%, -50%) translate(${edgeCenterX}px,${edgeCenterY}px)`,
-          }}
-        >
-          <EdgeButton onClick={onClick} />
-        </div>
-      </EdgeLabelRenderer>
+      {flowMode !== 1 ? (
```

Comment:
You can simplify the conditional rendering by using only the logical AND (&&) operator. This way, you won't need to include the unnecessary null.

What are your thoughts on this approach?